### PR TITLE
Handles pointer type for time.Time in "htmlDate"

### PR DESCRIPTION
**Pull request by mapreal19**

### CODE REVIEW

**Comment by mapreal19** [comment]
In Gorm sometimes is useful to have pointer types: http://gorm.io/docs/create.html#Default-Values

File: date.go [hunk 1]
```diff
@@ -28,6 +28,8 @@ func dateInZone(fmt string, date interface{}, zone string) string {
 		t = time.Now()
 	case time.Time:
 		t = date
+	case *time.Time:
```
Comment (mapreal19) [inline date.go]: If accepting this let me know if I should cover this with a unit test

Comment (mattfarina) [reply date.go]: This looks good. Can you please add a test.

Comment (mapreal19) [reply date.go]: Sure will do!

**Comment by mapreal19** [comment]
@mattfarina I'll open a new PR with tests when having time (I didn't forget about this)

**Pull request merged**